modules/openapi-generator/src/main/resources/kotlin-client/jvm-common/infrastructure/Serializer.kt.mustache

```diff
@@ -51,6 +52,8 @@ import java.util.concurrent.atomic.AtomicInteger
 import java.util.concurrent.atomic.AtomicLong
 {{/kotlinx_serialization}}
 
+{{#moshi}}import {{modelPackage}}.*{{/moshi}}
```

Comment:
Same as https://github.com/OpenAPITools/openapi-generator/pull/9159/files#r614518206

Reply:
Changed. Thanks for the hint :+1: